lib/operations/collection_ops.js

```diff
@@ -469,9 +468,9 @@ function findAndModify(coll, query, sort, doc, options, callback) {
     queryObject.sort = sort;
   }
 
-  queryObject.new = options.new ? true : false;
-  queryObject.remove = options.remove ? true : false;
-  queryObject.upsert = options.upsert ? true : false;
+  queryObject.new = options.new;
```

Comment:
FYI could be: `Object.assign(queryObject, { new: options.new, remove: options.remove, upsert: options.upsert });`

Reply:
Or could this even be a case for `decorate`?

Reply:
I don't think it's a good case for `decorate` because there are too many options that we don't want to add to `queryObject` here.

Reply:
I think `decorateCommand` can do what we want here, but it would be better if we just named the util function `pluckAndApply` or something